jujugui/static/gui/src/app/components/create-payment-user/create-payment-user.js

```diff
@@ -116,9 +110,9 @@ class CreatePaymentUser extends React.Component {
       name: address.name,
       email: refs.emailAddress.getValue(),
       addresses: [address],
-      vat: business && refs.VATNumber.getValue() || null,
+      vat: (business && refs.VATNumber.getValue()) || null,
```

Comment:
I don't get this one....

Reply:
See: https://prettier.io/blog/2018/05/27/1.13.0.html#insert-more-parentheses-in-math-expressions-4413-and-4407-by-duailibe